src/pe/mod.rs

```diff
@@ -3,7 +3,12 @@
 
 // TODO: panics with unwrap on None for apisetschema.dll, fhuxgraphics.dll and some others
```

Comment:
unrelated but i wonder if this comment still true

Reply:
That's a good question, I don't have those Windows binaries on my hand, but I can check them.